kymatio/scattering3d/backend/torch_backend.py

```diff
@@ -58,11 +58,126 @@ def compute_integrals(input_array, integral_powers):
     return integrals
 
 
+class Pad(object):
+    def __init__(self, pad_size, input_size, pre_pad=False):
+        """Padding which allows to simultaneously pad in a reflection fashion
+        and map to complex. 
+
+        Parameters
+        ----------
+        pad_size : list of 4 integers
+            Size of padding to apply [top, bottom, left, right].
+        input_size : list of 2 integers
+            Size of the original signal [height, width].
+        pre_pad : boolean, optional
+            If set to true, then there is no padding one simply adds the
+            imaginary part. 
+
+        """
+        self.pre_pad = pre_pad
+        self.pad_size = pad_size
+        self.input_size = input_size
+
+        self.build()
+
+    def build(self):
+        """Builds the padding module. 
+            
+            Attributes
+            ----------
+            padding_module : ReflectionPad2d
+                Pads the input tensor using the reflection of the input
```

Comment:
Are we reflection padding here? I think we need at least 0 padding as an option, if not as the default